lib/daemon/index.js

```diff
@@ -1,45 +1,56 @@
 "use strict";
 
-const tap = require("lodash/fp").tap;
 const cpus = require("os").cpus;
 const path = require("path");
 const workerpool = require("workerpool");
 
 const actions = require("../actions");
 const Cache = require("../utils/cache");
-const hash = require("../utils/hash");
 
 class InspectpackDaemon {
   /**
    * Start the daemon with an asynchronously initialized cache.
    *
    * @param {Object} opts                  Object options
-   * @param {Object} opts.cacheFilename    The filename of the daemon cache
+   * @param {Object} opts.cache            A cache instance
    * @returns {InspectpackDaemon} The daemon with cache
    */
   static create(opts) {
```

Comment:
Do you think it's worth keeping these factory methods throughout now that the constructor handles everything?

Reply:
Not sure... Maybe keep now because upstream (webpack-dashboard) uses...